pkg/router/template/configmanager/haproxy/manager.go

```diff
@@ -915,8 +928,23 @@ func (entry *routeBackendEntry) BuildMapAssociations(route *routeapi.Route) {
 	}
 }
 
+// validateBlueprint runs extended validation on a blueprint route.
+func validateBlueprintRoute(route *routeapi.Route) error {
```

Comment:
The function name in the comment and the actual function name differ.

Reply:
Ack will fix - thx.